lib/LaTeXML/Package/actuarialangle.sty.ltxml

```diff
@@ -0,0 +1,43 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  actuarialangle                                                     | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+InputDefinitions('actuarialangle', type => 'sty', noltxml => 1);
+
+DefConstructor('\actuarialangle{}',
+  "<ltx:XMApp>"
+    . "<ltx:XMTok role='ENCLOSE' enclose='actuarial' meaning='actuarial-angle' />"
+    . "<ltx:XMWrap>#1</ltx:XMWrap>"
+    . "</ltx:XMApp>");
+
+DefMath('\overanglebracket{}', "\x{23E0}", operator_role => 'OVERACCENT'); # TOP TORTOISE SHELL BRACKET
+
+Let('\group', '\overanglebracket');
+
+# preserve reversions
```

Comment:
by the way, if there is a better way to preserve the reversions, please let me know! I copied the idea from `\prescript` in `mathtools`

Reply:
If I'm understanding what you're trying to do, I think `DefMath` has some built-in magic for dealing with such cases.
Does the following replacement for the last two lines do what you want:
```
  DefMath('\group{}','\lx@actuarialangle@group{#1}');
```

Likewise the following could replace the loop body below:
```
  Let('\lx@actuarialangle@'.$cmd, '\\'.$cmd);
  DefMath('\\'.$cmd, '\lx@actuarialangle@'.$cmd);
```

Reply:
That works very well, I have updated the PR, thanks.